### PR TITLE
Resend jobs to queue on failure.

### PR DESCRIPTION
**Pull request by throughnothing**

### CODE REVIEW

File: lib/Sque/Job.pm [hunk 1]
```diff
@@ -33,6 +33,17 @@ has queue => (
 
 has args => ( is => 'rw', isa => 'ArrayRef', default => sub {[]} );
 
+has count => (
```
Comment (ironcamel) [inline lib/Sque/Job.pm]: Should probably name it something better than count. `retries`?

**Comment by ironcamel** [comment]
Do you need to reset `count` somewhere?

File: lib/Sque/Job.pm (hunk 1 above)
Comment (throughnothing) [inline lib/Sque/Job.pm]: Yeah, i wasn't sure about `count`...i like `retries`.

**Comment by throughnothing** [comment]
@ironcamel Not sure what you mean, why would I reset count?

For now, this PR will just retry indefinitely if something keeps failing.  What I want to add is a default of say 10 retries (configurable) or something, before it sends it to  `$job->queue . '-failed'` or something like that.  That way, it'll count the amount of times it retries, and at a certain threshold, send it to a separate failed queue that can be monitored/watched.

Thoughts?

**Comment by ironcamel** [comment]
On Wed, Jan 16, 2013 at 3:32 PM, William Wolf notifications@github.com wrote:

> @ironcamel Not sure what you mean, why would I reset count?
> 
> For now, this PR will just retry indefinitely if something keeps failing. What I want to add is a default of say 10 retries (configurable) or something, before it sends it to $job->queue . '-failed' or something like that. That way, it'll count the amount of times it retries, and at a certain threshold, send it to a separate failed queue that can be monitored/watched.
> 
> Thoughts?

Makes sense.

**Pull request closed without merging**